charts/gardener-extension-admission-calico/values.yaml

```diff
@@ -13,12 +13,12 @@ global:
   healthPort: 8081
   vpa:
     enabled: true
-    resourcePolicy:
-      minAllowed:
-        memory: 64Mi
-      maxAllowed:
-        cpu: 4
-        memory: 10G
+    resourcePolicy: {}
+#     minAllowed:
+#       memory: 64Mi
+#     maxAllowed:
+#       cpu: 4
+#       memory: 10G
```

Comment:
Why is this commented and not removed?

Reply:
It is still possible to set it by specifying it. The comments make it easy to see how to set it when deploying the extension/admission controller.